hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/OMDBInsightSearchEndpoint.java

```diff
@@ -389,4 +346,20 @@ private KeyEntityInfo createKeyEntityInfoFromOmKeyInfo(String dbKey,
     return keyEntityInfo;
   }
 
+  private boolean validateStartPrefix(String startPrefix) {
```

Comment:
This similar method is duplicated in `OMDBInsightEndpoint` class also. Given this comment earlier but not fixed.

Reply:
Done